Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -192,15 +194,19 @@ Ext.define('Zenoss.ReportTreePanel', {
 treesm = new Zenoss.TreeSelectionModel({
     listeners: {
         'selectionchange': function (sm, newnode) {
-            if (newnode === null) {
+            if (newnode === null || !report_panel.backcompat.processed
+                || !report_panel.graph_reports.graphs_added) {
                 return;
             }
             var attrs = newnode[0].data;
             if (attrs.isGraphReport) {
+                report_panel.graph_reports.graphs_added = false;
                 report_panel.renderGraphReport(attrs);
             } else if (attrs.isMultiGraphReport) {
+                report_panel.graph_reports.graphs_added = false;
                 report_panel.renderMultiGraphReport(attrs);
             } else {
+                report_panel.backcompat.processed = false;
```

Comment:
But where are you setting  `backcompat.processed` again to true? In [this](https://github.com/zenoss/zenoss-prodbin/pull/2091/files#diff-bb0add8e6b973593ece583f3eb114a48R148) load handler?